R/registry.R

```diff
@@ -202,6 +206,22 @@ create_registry <- function(cm, outpat, time = Sys.time()) {
   website_info$on_bioc <- purrr::map(website_info$name,
                                      get_bioc, bioc_names)
 
+  get_type <- function(status) {
```

Comment:
these "types" are the three tabs we have on https://roweb3-hugo.netlify.app/packages/all